keras_cv/losses/iou_loss.py

```diff
@@ -82,7 +82,9 @@ def call(self, y_true, y_pred):
                 f"the bounding boxes. Received y_true.shape[-1]={y_true.shape[-1]}."
             )
 
-        ious = bounding_box.compute_iou(y_true, y_pred, self.bounding_box_format)
+        ious = bounding_box.compute_iou(
+            y_true[..., :4], y_pred[..., :4], self.bounding_box_format
```

Comment:
why is it concatenated here? Shouldn't we expect dict tensors, or simply single Tensor at this point?

Reply:
Good point, updated to conform